Sources/Maker.swift

```diff
@@ -1016,3 +1001,16 @@ public final class Maker {
         }
     }
 }
+
+extension OptionSet where RawValue: BinaryInteger, Element == Self  {
+    public func forEach(_ body: (Self) -> Void) {
```

Comment:
What is the purpose of this `extension`?

Reply:
Without this extension I can't use switch in `edges` function and should use `if` chaining. Are you sure its better than elegant `switch` in cycle? + I use it in tests